src/screens/Feed/FilterDialog.tsx

```diff
@@ -42,44 +42,44 @@ const FeedScreenFilterDialog: VFC<FeedScreenFilterDialog> = ({
             style={filterDialogStyles.radioArea}
             onPress={() => setTargetFeedType(FeedTargetType.Normal)}
           >
-            <>
+            <React.Fragment>
```

Comment:
what is the difference between <> and <React.Fragment> ?

Reply:
There is no difference really. It was just an eslint error for this rule https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-fragments.md. 
We can enforce one of them to ensure we follow the same approach.

Reply:
as we had conversation in web team it's ok